CMakeLists.txt

```diff
@@ -268,24 +268,23 @@ if(SENTRY_WITH_LIBUNWINDSTACK)
 endif()
 
 if(SENTRY_BACKEND_CRASHPAD)
-	add_subdirectory(external/crashpad EXCLUDE_FROM_ALL)
-	target_include_directories(sentry PRIVATE
-		"$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/external/crashpad>"
-		"$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/external/crashpad/third_party/mini_chromium/mini_chromium>"
-	)
-	# we include a crashpad header, which itself needs C++14 support
-	target_compile_features(sentry PUBLIC cxx_std_14)
```

Comment:
can you also check the unit test file? The pytest suite does not build the unittests with crashpad, thats why CI will not complain about this. But we had problems when building the unittests with the crashpad backend manually. does that work correctly?

Reply:
What option combination is failing?
Please open a pr with the (failing) test, then I can take a look.

Reply:
just doing a `cmake -B build && cmake --build build --target sentry_test_unit`, which uses the crashpad backend by default. But I think now all the flags are correctly propagated through the dependency chain.

Reply:
I just tried and current master (before this pr) is working for me, so there must be another problem you are seeing.